scripts/setup_firebase.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/node
+
+/***
+ * This script will populate test entries in your firebase firestore for use for developing on the findadoc app.
+ * See details in README.md.
+ * */
+
+require("dotenv").config({path: __dirname + '/../.env'});
+const fs = require('fs').promises;
+const firebase = require("firebase");
+require('firebase/firestore');
+
+const FIXTURES_DIR = __dirname + '/fixtures/';
+
+const fixtures = [
+    {
+        "collection" : "clinics",
+        "file" : FIXTURES_DIR + 'clinics_collection_sample.json'
+    },
+    {
+        "collection" : "pending",
+        "file" : FIXTURES_DIR + 'pending_collection_sample.json'
+    },
+    {
+        "collection" : "reports",
+        "file" : FIXTURES_DIR + 'reports_collection_sample.json'
+    },
+    {
+        "collection" : "users",
+        "file" : FIXTURES_DIR + 'users_collection_sample.json'
+    }
+];
+
+const initialize = () => {
+    console.log(`Initializing using firebase API key ${process.env.FIREBASE_API_KEY}`)
+    firebase.initializeApp({
+        apiKey: process.env.FIREBASE_API_KEY,
+        authDomain: process.env.FIREBASE_AUTH_DOMAIN,
+        projectId: process.env.FIREBASE_PROJECT_ID
+    });
+}
+
+const loadFixture = async (filepath) => {
+    try {
+        const json = await fs.readFile(filepath, "utf-8");
+        return JSON.parse(json);
+    }
+    catch (e) {
+        console.log(e.message);
+    }
+}
+
+const storeToFirestore = async (collectionData, collectionName) => {
+    try {
+        const db = firebase.firestore();
+        let count = 0;
+        for (let item of collectionData) {
+            let docRef = await db.collection(collectionName).add(item);
+            console.log(`Successfully persisted to ${collectionName} as: ${docRef.id}`);
+            count++;
+        }
+        console.log(`Successfully persisted ${count} documents to ${collectionName}.`);
+    } catch (e) {
+        console.log(e.message);
+    }
+}
+
+const isProd = () => {
+    let productionResourcesUsed = process.env.FIREBASE_PROJECT_ID == "findadoc-bc230" || process.env.FIREBASE_DATABASE_URL == "https://findadoc-bc230-default-rtdb.firebaseio.com";
+    if(productionResourcesUsed) {
+        console.log("You're currently using the production database. This script will exit with no changes to avoid data loss.")
+    }
+    return productionResourcesUsed;
+}
+
+const main = async () => {
+    if (isProd()) { return }
+    initialize();
+    for (let fixture of fixtures) {
+        let records = await loadFixture(fixture.file);
+        storeToFirestore(records, fixture.collection);
+    }
+}
+
+main();
```

Comment:
need to await main

Reply:
That won't work outside the context of an `async` function, and results in a syntax error:

`SyntaxError: await is only valid in async function`

Reply:
You can. I've used it a couple times before.
see [this thread ](https://stackoverflow.com/a/56590390/3417143) for more examples

 (granted I'm not sure our version of Nuxt supports top level `await` but you should be able to Promise.resolve()

Reply:
This script doesn't run in nuxt, but rather using node.js directly. Since that's a very recently added feature I think it will kill compatibility for not much benefit?

Reply:
Sidestepped this with an anonymous wrap. This work for you @Ermish ?

Reply:
that works! thanks for being patient 🥇

Reply:
Huzzah